pom.xml

```diff
@@ -102,15 +124,21 @@
             <version>2.0</version>
         </dependency>
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>junit</artifactId>
+            <version>1.12</version>
+            <scope>test</scope>
```

Comment:
Doesn't seem so, the plugin wasn't loaded when the unit tests ran, so I had to add it explicitly.